<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TorrentWindow</class>
 <widget class="QMainWindow" name="TorrentWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TorrentWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QTreeWidget" name="torrentListTreeWidget">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <column>
         <property name="text">
          <string notr="true">#</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Size</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Progress</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Completion</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="torrentToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd"/>
   <addaction name="actionStartTorrent"/>
   <addaction name="actionStopTorrent"/>
   <addaction name="separator"/>
   <addaction name="actionRemoveTorrent"/>
   <addaction name="actionSettings"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <action name="actionStartTorrent">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/start.png</normaloff>:/icons/icons/start.png</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="toolTip">
    <string>Start Torrent</string>
   </property>
  </action>
  <action name="actionStopTorrent">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/stop.png</normaloff>:/icons/icons/stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="toolTip">
    <string>Stop Torrent</string>
   </property>
  </action>
  <action name="actionRemoveTorrent">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/trash.png</normaloff>:/icons/icons/trash.png</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="toolTip">
    <string>Remove Torrent</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/settings.png</normaloff>:/icons/icons/settings.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="toolTip">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/add.png</normaloff>:/icons/icons/add.png</iconset>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
   <property name="toolTip">
    <string>Add New Torrent</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
